app/src/main/java/org/p2p/wallet/moonpay/clientsideapi/MoonpayClientSideApi.kt

```diff
@@ -45,6 +46,6 @@ interface MoonpayClientSideApi {
         @Path("token") tokenSymbol: String,
         @Query("apiKey") apiKey: String,
         @Query("quoteCurrencyCode") fiatName: String,
-        @Query("baseCurrencyAmount") tokenAmount: Double
+        @Query("baseCurrencyAmount") tokenAmount: BigDecimal
```

Comment:
when parsing from the server, BigDecimal can be parsed very strangely  
baseCurrencyAmount: 0.15
but Gson will parse it as 0.149999999999999